twitter.css

```diff
@@ -0,0 +1,110 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.twitter {
+  display: grid;
+  grid-template:
+    "left main search" /
+    1fr 2fr 1fr;
```

Comment:
Consider fixing the width of the the left and search area and only using fractions for the main part.